packages/ui/src/stories/formkit/Search.stories.ts

```diff
@@ -0,0 +1,22 @@
+import { StoryFn } from '@storybook/vue3'
```

Comment:
Maybe it makes sense to have a story for all pass-through input types, as there's just too many and we're not doing anything much for decoration, rather we're defining pass-through does happen.